stubs/django/db/models/query.pyi

```diff
@@ -3,8 +3,119 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import Any
+from typing import (
+    Any,
+    Collection,
+    Generic,
+    Sized,
+    Reversible,
+)
+
+from django.db.models.manager import Manager
+from django.db.models.base import Model
+
+_T = TypeVar("_T", bound=models.Model, covariant=True)
 
 def __getattr__(name: str) -> Any: ...
 
-# Placeholder stub.
+class QuerySet(Generic[_T], Collection[_T], Reversible[_T], Sized):
+    model: Any = ...
+    def __init__(
+        self,
+        model: Optional[Any] = ...,
```

Comment:
Note that `Optional[Any]` is the same as `Any` since `Any` can be `None`

Reply:
Thank you very much! I will now do the changes like that

Reply:
The "Optional[Any]" here was just for illustration, to emphasize that the parameter is optional. So this was just for a human readability. But since .pyi files are not supposed to be read by humans often, and it is sometimes better to read source file or the documentation, I will replace the "Optional[Any]" to "Any".